### ISSUE TITLE
No module named 'adaptive_conv_cuda_impl' and 'adaptive_conv_cpp_impl'

### ISSUE THREAD

**Issue by Hello123hello123** [mhamilton723/FeatUp#77]
When I tried to reproduce the code, I found that 'adaptive_comvv_cuda_impl' and 'adaptive_comvv_cpp_impl' could not be imported. Can you tell me how to solve this problem？

**Comment by HomoBirraiolus** [comment]
yup this also prevents me from running the model, so basically dot for now.
I will try to further investigate if something is missing betwen my graphic card, Cuda version and pytorch version. So far i am stuck

**Comment by bobiblazeski** [comment]
I had a same issue when  when using 

> pip install git+https://github.com/mhamilton723/FeatUp

Downloaded the repo first and then installed , seems to work well now

> git clone https://github.com/mhamilton723/FeatUp.git
> cd FeatUp
> pip install -e .

**Comment by ShiJinghao566** [comment]
> 是的，这也阻止了我运行模型，所以现在基本上是点。我将尝试进一步调查我的显卡、Cuda 版本和 pytorch 版本之间是否缺少某些内容。到目前为止我被卡住了

Please ask if you solved it, so far I still have a message that this file is missing

**Comment by YANGFENG0001** [comment]
也许是没有构建cuda扩展，需要运行setup.py进行构建：
```python
python setup.py build_ext --inplace
```
